Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection_test.py

```diff
@@ -127,8 +127,8 @@ def test_get_machine_investigation_package_command(mocker):
     from MicrosoftDefenderAdvancedThreatProtection import get_machine_investigation_package_command
     mocker.patch.object(client_mocker, 'get_investigation_package', return_value=INVESTIGATION_PACKAGE_API_RESPONSE)
     mocker.patch.object(atp, 'get_machine_action_data', return_value=INVESTIGATION_ACTION_DATA)
-    result = get_machine_investigation_package_command(client_mocker, {'machine_id': '123', 'comment': 'test'})
-    assert result.outputs['MicrosoftATP.MachineAction(val.ID === obj.ID)'] == INVESTIGATION_ACTION_DATA
+    _, res, _ = get_machine_investigation_package_command(client_mocker, {'machine_id': '123', 'comment': 'test'})
+    assert res['MicrosoftATP.MachineAction(val.ID === obj.ID)'] == INVESTIGATION_ACTION_DATA
 
```

Comment:
Please revert this changes as mention in comment in 
`get_machine_investigation_package_command`